pkg/registry/sync/sync.go

```diff
@@ -42,7 +45,21 @@ const (
 	defaultPort = "5000"
 )
 
-func ToRegistry(ctx context.Context, sys *types.SystemContext, src, dst string, reportWriter io.Writer, selection copy.ImageListSelection) error {
+type Options struct {
```

Comment:
make fields more clear, like SystemContext, ReportWriter, FastError could be OmitError?